i18n/README.md

```diff
@@ -2,10 +2,23 @@
 
 Thank you for considering adding an additional translation to Video Hub App!
 
+# Setup
 Translations take seconds using the _Google Translation API_ and the [translate-json-object](https://github.com/KhaledMohamedP/translate-json-object) library, but it does require a _Google Cloud API_ key (please use your own).
 
+### How to get Google Could API Key
```

Comment:
Spelling mistake: *Could* should be *Cloud*.

Reply:
Thank you for the catch!